alexandria/core/thumbs.py

```diff
@@ -45,6 +46,13 @@ def create_thumbnail(file):
     )
     etag = client.put_object(path_to_preview_image, thumb_file.object_name)
 
+    # remove locked file created by PreviewManager.get_jpeg_preview()
+    # introduced with preview-generator version 0.14
+    # https://github.com/algoo/preview-generator/commit/e8c16499ff38c21ab5a65110fc29635520674ec7#
```

Comment:
I'd put this comment into the commit message since it has no value in the code per se but only in the history of the code..